<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fireball</class>
 <widget class="QWidget" name="fireball">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>fireball</string>
  </property>
  <widget class="QTextBrowser" name="outputConsole">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>30</y>
     <width>611</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic</family>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'URW Gothic'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Noto Sans'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="markdown" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>940</y>
     <width>611</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="onButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Click this button to start tracking process</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 25pt &quot;URW Bookman&quot;;
font: 87 28pt &quot;Noto Serif CJK SC&quot;;</string>
      </property>
      <property name="text">
       <string>Start 启动</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="offButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Click this botton to kill tracking process</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 28pt &quot;Noto Serif CJK SC&quot;;</string>
      </property>
      <property name="text">
       <string>Stop 中止</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>620</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCheckBox" name="schedulerON">
      <property name="baseSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Timer</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>URW Gothic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>8</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeEdit_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>URW Gothic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>20</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>960</y>
     <width>1091</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="boxButton_1">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 24pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Cage 1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="boxButton_2">
      <property name="styleSheet">
       <string notr="true">font: 63 24pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Cage 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="boxButton_3">
      <property name="styleSheet">
       <string notr="true">font: 63 24pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Cage 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="boxButton_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 24pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Cage 4</string>
      </property>
      <property name="iconSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1200</width>
     <height>900</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman</family>
     <pointsize>40</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Waiting for image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1680</x>
     <y>780</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;URW Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
